lib/src/commons/BrowserUtils.d.ts

```diff
@@ -228,4 +228,27 @@ export declare namespace BrowserUtils {
      * @param expectedText expected alert's text
      */
     function expectAlertText(expectedText: string): void;
+    /**
+     *
+     * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+     */
+    function mouseButtonDown(mouseButton?: string): void;
+    /**
+     * @param selector - element to move to, If not specified or is null, the offset is relative to current position of the mouse.
+     * @param xOffset - X offset to move to, relative to the top-left corner of the element If not specified, the mouse will move to the middle of the element.
```

Comment:
`If not specified ... to the middle of the element`. Which element?

Reply:
Documentation is not saying anything more specific i assume in case of offset then you are obligated to have an element selector